starcompactor/extractors/machine.py

```diff
@@ -93,19 +87,19 @@ def get_machine_event_baremetal(mysql_args, tmp_path, tmp_sql_file_name):
     tmp_ironic_node_table_sql_file_name = 'ironic_nodes_{}'.format(tmp_sql_file_name)
     process_extract_node_table = subprocess.Popen("sed -n -e '/DROP TABLE.*`nodes`/,/UNLOCK TABLES/p' {} > {}".format(os.path.join(tmp_path, tmp_ironic_database_sql_file_name), os.path.join(tmp_path, tmp_ironic_node_table_sql_file_name)), shell=True)
     process_extract_node_table.wait()
-    process_ironic = subprocess.Popen('mysql --user={} --password={} --init-command="SET SESSION FOREIGN_KEY_CHECKS=0;" --one-database {} < {}'.format(mysql_args['user'], mysql_args['passwd'], ironic_tmp_database_name, os.path.join(tmp_path, tmp_ironic_node_table_sql_file_name)), shell=True)
+    process_ironic = subprocess.Popen('mysql --user={} --password={} --host {} --port {} --init-command="SET SESSION FOREIGN_KEY_CHECKS=0;" --one-database {} < {}'.format(mysql_args['user'], mysql_args['passwd'], mysql_args['host'], mysql_args['port'], ironic_tmp_database_name, os.path.join(tmp_path, tmp_ironic_node_table_sql_file_name)), shell=True)
```

Comment:
How was this working before?

Reply:
If not specified, it will read the defaults from the files like ".my.cnf".